ssz/hash.py

```diff
@@ -1,10 +1,12 @@
+import functools
 import hashlib
 
 from eth_typing import (
     Hash32,
 )
 
 
+@functools.lru_cache(maxsize=2**30)
```

Comment:
This seems like a very large cache.  Ignoring the input data, I think this is 32 GB worth of hashes....

Reply:
Yeah! That's a stub now for benchmarking the limitation of this approach for eth2 usecases.